### PR TITLE
#2018 - Patroni update on Dev Test and Prod, DB-migration revert enable in API

### PR DESCRIPTION
**Pull request by guru-aot**
- Removed the fields in the service definition, which does not allow the connections of other pods to the stateful set.
-  Enable db-migration scripts to run on atleast one pod when revert is needed, so enabled it in API pod

### CODE REVIEW

File: devops/openshift/database/patroni-deploy.yml [hunk 1]
```diff
@@ -31,10 +31,6 @@ objects:
       role: master
       cluster-name: ${NAME}
       app.kubernetes.io/name: patroni
-      sessionAffinity: None
```
Comment (andrewsignori-aot) [inline devops/openshift/database/patroni-deploy.yml:34]: I notice that these configs were not present in the past either, nice catch.
Was it tested on DEV?

**Review by andrewsignori-aot — APPROVED** [review]
Nice catch, looks good 👍

**Review by ann-aot — APPROVED** [review]
👍

**Review by dheepak-aot — APPROVED** [review]
👍

File: devops/openshift/database/patroni-deploy.yml (hunk 1 above)
Comment (andrepestana-aot) [inline devops/openshift/database/patroni-deploy.yml:34]: Is those properties the cause the problem to not work and need to do some commands for this to have a main instance?

**Review by andrepestana-aot — APPROVED** [review]

File: devops/openshift/database/patroni-deploy.yml (hunk 1 above)
Comment (sh16011993) [reply devops/openshift/database/patroni-deploy.yml:34]: So basically removing these properties allows for connection with services outside this cluster?

Comment (sh16011993) [inline devops/openshift/database/patroni-deploy.yml:34]: So basically removing these properties allows for connection with services outside this cluster?

**Review by sh16011993 — APPROVED** [review]
👍

File: sources/packages/backend/apps/api/Dockerfile [hunk 2]
```diff
@@ -15,6 +15,7 @@ RUN npm ci
 # Copying sources.
 COPY ./apps/api ./apps/api
 COPY ./libs ./libs
+COPY ./apps/db-migrations ./apps/db-migrations
```
Comment (dheepak-aot) [inline sources/packages/backend/apps/api/Dockerfile:18]: 👍

**Pull request merged**